### DISCUSSION TITLE
`nativeElement` on Grid

### DISCUSSION THREAD

**Discussion by baleeds** [IgniteUI/igniteui-angular#10809] [Q&A]
Previously on the IgxCell, there was a property to retrieve the `nativeElement`.  That has been removed in the `IgxGridCell` of version 12.  Is there an alternative method to get the underlying DOM element?  We are using it as the target for an IgxOverlay and to check for custom classes on the cell element.

**Comment by dkamburov** [comment]
Ideally there should be no need for the `nativeElement` of the cell.

How about using the cell template inside the column declaration to achieve this scenario?
You should be able to add a container for the cell value and use that container as a target of the IgxOverlay.

Not sure how you achieve the custom classes, but you might be able to check the condition from the cell or row context or use the `cellStyles` of the column.
